src/Microsoft.Graph/Requests/Generated/UserDrivesCollectionResponse.cs

```diff
@@ -13,13 +13,13 @@ namespace Microsoft.Graph
     /// <summary>
     /// The type UserDrivesCollectionResponse.
     /// </summary>
-    [DataContract]
+    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
     public class UserDrivesCollectionResponse
     {
         /// <summary>
         /// Gets or sets the <see cref="IUserDrivesCollectionPage"/> value.
         /// </summary>
-        [DataMember(Name = "value", EmitDefaultValue = false, IsRequired = false)]
+		[JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName ="value", Required = Required.Default)]
```

Comment:
These changes were added to 1) improve code consistency between DataMember/JsonProperty and 2) to conform to .NETStandard 1.1. So it is unnecessary to make these changes, since there is no functional difference between the two. It is simply a matter of whether you want to head towards .NETStandard in the future.